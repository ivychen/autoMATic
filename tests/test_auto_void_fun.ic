auto func() {
    printstr("func is a void-returning function");
}

int main() {
    func();
    return 0;
}
