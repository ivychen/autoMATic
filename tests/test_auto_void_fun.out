func is a void-returning function
